plugins/notifiers/slack/client.go

```diff
@@ -51,25 +49,25 @@ type Config struct {
 //go:embed templates/*
 var defaultTemplates embed.FS
 
-func New(config *Config) *notifier {
-	return &notifier{
+func NewNotifier(config *Config, slackIDCache map[string]string, httpClient utils.HTTPClient) *Notifier {
```

Comment:
can we exclude `slackIDCache` from `NewNotifier` parameter? this should be private for `Notifier` only

Reply:
Done